src/desktop/space/mod.rs

```diff
@@ -193,11 +193,17 @@ impl Space {
     /// The scale is the what is rendered for the given output
     /// and may be fractional. It is independent from the integer scale
     /// reported to clients by the output.
+    ///
+    /// *Note:* Remapping an output does reset it's damage memory.
```

Comment:
So, does that mean the output will be considered as 100% damaged, or 0% damaged?

Reply:
100% damaged.
without a memory the output assumes everything is new.